### PR TITLE
Paginate the Article

### PR DESCRIPTION
**Pull request by noor848**
Conduit #17

### CODE REVIEW

File: Conduit/Conduit/Controllers/ArticleController.cs [hunk 1]
```diff
@@ -78,12 +77,12 @@ public async Task<ActionResult<List<Article>>> GetAllArticle(int PageNumber)
                 var ArticlesNyumberInPage = 3f;
```
Comment (hayatzsala) [inline Conduit/Conduit/Controllers/ArticleController.cs]: it should be const, not var
* typo in Number
* not float should be int, it's number of articles

File: Conduit/Conduit/Controllers/ArticleController.cs [hunk 2]
```diff
@@ -78,12 +77,12 @@ public async Task<ActionResult<List<Article>>> GetAllArticle(int PageNumber)
...
                 var pageCount = Math.Ceiling(_context.Articles.Count() / ArticlesNyumberInPage);
 
-                var articles = await _ArticlesRepositry.GetAllArticlePaginated(PageNumber, ArticlesNyumberInPage);
+                var article = await _ArticlesRepositry.GetAllArticlePaginated(PageNumber, ArticlesNyumberInPage);
```
Comment (hayatzsala) [inline Conduit/Conduit/Controllers/ArticleController.cs:80]: why you changed the article name to singular?  articles is better,

File: Conduit/Conduit/Controllers/ArticleController.cs [hunk 3]
```diff
@@ -78,12 +77,12 @@ public async Task<ActionResult<List<Article>>> GetAllArticle(int PageNumber)
                 var ArticlesNyumberInPage = 3f;
                 var pageCount = Math.Ceiling(_context.Articles.Count() / ArticlesNyumberInPage);
 
-                var articles = await _ArticlesRepositry.GetAllArticlePaginated(PageNumber, ArticlesNyumberInPage);
```
Comment (hayatzsala) [inline Conduit/Conduit/Controllers/ArticleController.cs:81]: hmmm,  I see the changes were merged before in main right?, it's not clear changes in PR

Comment (noor848) [reply Conduit/Conduit/Controllers/ArticleController.cs:81]: sorry i i was need the update from main that why merge one of updates

File: Conduit/Conduit/Controllers/ArticleController.cs (hunk 2 above)
Comment (noor848) [reply Conduit/Conduit/Controllers/ArticleController.cs:80]: there was conflict when i pulled the update but ill change it back to the previous update

**Pull request merged**